pkg/katautils/config.go

```diff
@@ -163,6 +165,16 @@ func (h hypervisor) path() (string, error) {
 	return ResolvePath(p)
 }
 
+func (h hypervisor) ctlpath() (string, error) {
+	p := h.CtlPath
+
+	if h.CtlPath == "" {
+		p = defaultHypervisorCtlPath
```

Comment:
Does this have any effect? I see that the `defaultHypervisorCtlPath` is always an empty string unless I missed a code path where this gets reassigned.

Reply:
Only acrn for now uses defaultHypervisorCtlPath. Shall I set this to /usr/bin/acrnctl? or do you want me to remove the empty check?

Reply:
Yes, it makes sense to set the default to /usr/bin/acrnctl.

Reply:
done.